src/plugins/imagery/components/Compass/CompassHUD.vue

```diff
@@ -99,24 +98,27 @@ export default {
             type: Number,
             required: true
         },
+        /* only useful if combined with tilt to find horizon
         roll: {
             type: Number,
             default: undefined
         },
+        */
         sunHeading: {
             type: Number,
             default: undefined
         },
-        cameraFieldOfView: {
+        cameraAngleOfView: {
             type: Number,
             default: undefined
         },
         cameraPan: {
             type: Number,
-            default: undefined
+            required: true
         }
     },
     computed: {
+        /* only useful if combined with tilt to find horizon
```

Comment:
Let's remove this commented code, we have it in version control if we need it in future.